### DISCUSSION TITLE
Enregistrer le formuaire apres chaque page

### DISCUSSION THREAD

**Discussion by anistouati** [MTESSDev/FRW#294] [Annonces]
Bonjour, 

Est-ce que c'est possible d'enregistrer les réponses aux questions par page? en cliquant sur le bouton Enregistrer.

Merci!

**Comment by LarcM7** [comment]
Bonjour, 

Oui, les réponses aux questions s'enregistrent sur le clic du bouton Enregistrer (ainsi qu'à chaque navigation de section).

Toutefois les pages ne font pas partie des données, on ne conserve aucune information sur quelle donnée est dans quelle page.

Il vous serait toujours possible de vous faire un groupe "Page1" dans la config de votre formulaire et de mettre les questions de cette page à l'intérieur, et ainsi de suite pour chaque page. 

https://formulaires.it.mtess.gouv.qc.ca/fr/Form/7/P700U/0/N/#p=sectionGroup

Toutefois, ce n'est pas une façon de faire que nous recommandons car vous ne pourrez plus mettre de groupes dans votre formulaire, car un groupe dans un groupe n'est pas supporté.

Pourquoi voulez-vous enregistrer les réponses "par page"?

Merci et bonne journée.

**Comment by anistouati** [comment]
La demande initiale était plutôt de faire les validations des champs requis dans chaque page (quand on clique sur Suivant)

**Reply by LarcM7** [reply]
ah ok 🙃

Non ce n'est pas possible. 

Merci et bonne journée.
